src/main/java/net/sf/jabref/importer/fetcher/ScienceDirectFetcher.java

```diff
@@ -114,20 +113,16 @@ public boolean processQuery(String query, ImportInspector dialog, OutputPrinter
      */
     private static List<String> getCitations(String query) throws IOException {
         String urlQuery;
-        ArrayList<String> ids = new ArrayList<>();
-        try {
-            urlQuery = ScienceDirectFetcher.SEARCH_URL + URLEncoder.encode(query, StandardCharsets.UTF_8.name());
-            int count = 1;
-            String nextPage;
-            while (((nextPage = getCitationsFromUrl(urlQuery, ids)) != null)
-                    && (count < ScienceDirectFetcher.MAX_PAGES_TO_LOAD)) {
-                urlQuery = nextPage;
-                count++;
-            }
-            return ids;
-        } catch (UnsupportedEncodingException e) {
```

Comment:
Does the caller of getCitations somewhere catches the runtimeException? If yes, then this should be adopted too.